auto sync 2022-03-03

nunnatsa opened the pull request:
auto sync upstream 2022-03-03

ravidbro commented:
the image build failed

bardielle commented:
@ravidbro I know - it's not the right solution for upgrading the machine-api-operator
I'm closing this PR

nirarg committed:
Change OWNERS file

nunnatsa committed:
Automatic sync with upstream

This PR adds a new github action to daily sync with upstream. It checks if there are new changes in upstream, and if there are, it creates a PR to sync.

The new PR is created by hard reseting to the upstream main branch, then restoring the downstream modified files.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>
